source/core/serial_solid.cc

```diff
@@ -39,8 +39,6 @@
 #include <deal.II/numerics/solution_transfer.h>
 #include <deal.II/numerics/vector_tools.h>
 
-#include <deal.II/particles/data_out.h>
-
 #include <boost/archive/text_iarchive.hpp>
 #include <boost/archive/text_oarchive.hpp>
```

Comment:
CLion shows me a lot of grey include meaning some of them may be used in this class. Are all the current includes used ?

Reply:
Most likely not. I can't spot them from my editor regrettably,.